generators/server/cleanup.ts

```diff
@@ -199,4 +199,10 @@ export default function cleanupOldServerFilesTask(this: BaseGenerator, taskParam
       this.removeFile(`${application.javaPackageSrcDir}security/oauth2/OAuthIdpTokenResponseDTO.java`);
     }
   }
+
+  if (this.isJhipsterVersionLessThan('8.1.0')) {
```

Comment:
Should be next possible release
```suggestion
  if (this.isJhipsterVersionLessThan('8.1.1')) {
```